### ISSUE TITLE
Hash-URL is not updated on OSM.org

### ISSUE THREAD

**Issue by tordans** [openstreetmap/iD#8114]
The configuration hash on OSM.org is not updated anymore. For example, custom image URL or the info "show mapillary" is not added to the URL.

https://www.openstreetmap.org/edit#map=19/52.94899/14.10464

Tested in Chrome + Chrome Canary.

--- 

Looking into this:
* https://github.com/openstreetmap/iD/issues/6492 about this feature as well
* The code that updates the location-hash-part still works
* The code that adds the params like source is still there at https://www.openstreetmap.org/assets/edit/id-60c5d3261020229458760d5c4a399f4c1988e5ca0a2a2fd17e923614b70a15f3.js which is included in https://www.openstreetmap.org/edit#map=19/52.47232/13.44548

**Comment by willemarcel** [comment]
@tordans do you need those params on the url to share an iD config with someone? I prefer to have a cleaner URL. Maybe a solution would be a share URL function to get the URL with alll the hash params.

**Comment by tordans** [comment]
@willemarcel yes, the state of the app config should be shareable via URL. I am pretty shure it was before. IMO, longer urls are not unclean - it is how the web works. Hiding the config in a config hash breaks the old system, makes it less hackable and more complex to maintain, IMO.
